storm-server/src/main/java/org/apache/storm/utils/ServerUtils.java

```diff
@@ -1309,8 +1309,6 @@ public static boolean isAnyPosixProcessPidDirAlive(Collection<Long> pids, String
                         + "likely pid {} was reused for a new process for expectedUser {}",
                         pidDir, actualUser, expectedUser, pid, actualUser);
```

Comment:
Looks like the log should be `a new process for actualUser {}"`

And is it possible to print out the cmdline of this processId so we know what the process really is? It seems unlikely that a processId is reused by another user (or root) so quickly. I hope to get more details when this happens. Thanks

Reply:
fixed log